cluster-provision/gocli/cmd/ssh.go

```diff
@@ -35,10 +36,25 @@ func ssh(cmd *cobra.Command, args []string) error {
 	}
 
 	// TODO we can do the ssh session with the native golang client
-	exitCode, err := docker.Terminal(cli, prefix+"-"+node, append([]string{"ssh.sh"}, args[1:]...), os.Stdout)
-	if err != nil {
-		return err
+	container := prefix + "-" + node
+	ssh_command := append([]string{"ssh.sh"}, args[1:]...)
+	file := os.Stdout
+	if terminal.IsTerminal(int(file.Fd())) {
```

Comment:
hm if you the one who define the file to
`file := os.Stdout` how it can be different from `terminal.IsTerminal(int(file.Fd()))`

Reply:
What do you mean ?

Reply:
ah now I see the `file.Fd()` changes when you connect with and without TTY